### ISSUE TITLE
CAPTCHAs wont show up due to update on 4chan's end

### ISSUE THREAD

**Issue by BlockJuice** [zixaphir/appchan-x#569]
Pulled from the 4chan announcement:
We just changed CAPTCHAs to show on-demand (once you start typing a post) for two reasons:
1) It should decrease their difficulty since Google penalizes you for requesting a lot of verification images, but not solving them, which had been happening.
2) For the privacy concerned among us, we still load the JS but it only hits the API when you're actually posting.
tl;dr: The number of black inkblot CAPTCHAs displayed should decrease dramatically.

Pic: 
![captcha](https://f.cloud.github.com/assets/1514318/2120714/f5d486ea-91c0-11e3-8aec-bebad7104f2a.png)

EDIT: Somebody posted a fix on 4chan that works for now. I posted it in another thread.
https://github.com/zixaphir/appchan-x/issues/564#issuecomment-34584171
